platform/ios/PolyPodApp/PolyPod/HomeScreenView.swift

```diff
@@ -0,0 +1,653 @@
+            
+            VStack(alignment: .leading, spacing: HomeScreenConstants.BigTile.textVerticalSpacing) {
+                Text(card.title)
+                    .foregroundColor(foregroundColor)
+                    .font(HomeScreenConstants.BigTile.title.font)
+                    .multilineTextAlignment(HomeScreenConstants.BigTile.title.alignment)
+                Text(card.description)
+                    .foregroundColor(foregroundColor)
+                    .font(HomeScreenConstants.BigTile.description.font)
+                    .multilineTextAlignment(HomeScreenConstants.BigTile.description.alignment)
+            }
+        }
+        .padding(HomeScreenConstants.BigTile.padding)
+        .frame(width: sizes.bigTileWidth,
+               height: sizes.bigTileWidth)
+        .background(card.backgroundColor)
+        .cornerRadius(HomeScreenConstants.Tile.cornerRadius)
+        .onTapGesture {
+            onFeatureSelected(card.id)
+        }
+    }
+}
+
+struct MediumCardView: View {
+    @Environment(\.homeScreenTileSizes) var sizes
+    @Environment(\.homeScreenFeatureSelected) var onFeatureSelected
+    
+    let card: Card
+    private let foregroundColor: Color
+    
+    init(card: Card) {
+        self.card = card
+        self.foregroundColor = card.backgroundColor.isLight ? .black : .white
+    }
+    
+    var body: some View {
+        HStack(spacing: HomeScreenConstants.MediumTile.horizontalSpacing) {
+            Image(uiImage: card.image)
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: sizes.smallTileWidth,
+                       height: sizes.smallTileWidth,
+                       alignment: .center)
+            VStack(alignment: .leading, spacing: HomeScreenConstants.MediumTile.textVerticalSpacing) {
+                Text(card.title)
+                    .foregroundColor(foregroundColor)
+                    .font(HomeScreenConstants.MediumTile.title.font)
+                    .multilineTextAlignment(HomeScreenConstants.MediumTile.title.alignment)
+                Text(card.description)
+                    .foregroundColor(foregroundColor)
+                    .font(HomeScreenConstants.MediumTile.description.font)
+                    .multilineTextAlignment(HomeScreenConstants.MediumTile.description.alignment)
+            }
+            .padding([.top, .bottom], HomeScreenConstants.MediumTile.textTopBottomPadding)
+            .padding([.trailing], HomeScreenConstants.MediumTile.textTrailingPadding)
+            Spacer(minLength: 0.0)
+        }
+        .frame(width: sizes.mediumTileWidth, height: sizes.smallTileWidth)
+        .background(card.backgroundColor)
+        .cornerRadius(HomeScreenConstants.Tile.cornerRadius)
+        .onTapGesture {
+            onFeatureSelected(card.id)
+        }
+    }
+}
+
+struct SmallCardView: View {
```

Comment:
Again, a good amount of boilerplate here... Probably no good (or fast) way to reduce it, but still.

Reply:
In initial iteration we went with what was faster and easier to grasp, developing the components separately instead of creating one with dynamic layout. But yeah, this could be improved.

Reply:
If it is simple, I suggest we keep it that way. Boilerplate is tedious but in this case not evil. Over engineering is a greater danger than boilerplate ♟️